test/optimus/pr/ValidateOptimus.wdl

```diff
@@ -143,16 +188,16 @@ task ValidateMetrics {
         fail=false
 
         if [ "$gene_metric_hash" == "${expected_gene_metric_hash}" ]; then
-            echo Computed and expected gene metrics match \( $gene_metric_hash \)
+            echo Computed and expected gene metrics match \( "$gene_metric_hash" \)
         else
-            echo Computed \( $gene_metric_hash \) and expected \( ${expected_gene_metric_hash} \) gene checksums do not match
+            echo Computed \( "$gene_metric_hash" \) and expected \( ${expected_gene_metric_hash} \) gene checksums do not match
             fail=true
         fi
 
         if [ "$cell_metric_hash" == "${expected_cell_metric_hash}" ]; then
-            echo Computed and expected cell metrics match \( $cell_metric_hash \)
+            echo Computed and expected cell metrics match \( "$cell_metric_hash" \)
         else
-            echo Computed \( $cell_metric_hash \) and expected \( ${expected_cell_metric_hash} \) cell metrics hashes do not match
+            echo Computed \( "$cell_metric_hash" \) and expected \( ${expected_cell_metric_hash} \) cell metrics hashes do not match
```

Comment:
Why is `cell_metric_hash` quoted and `expected_cell_metric_hash` not?

Reply:
done